src/app/core/models/attendance-sheets-interface.model.ts

```diff
@@ -0,0 +1,23 @@
+// export interface AttendanceSheet {
```

Comment:
Please remove commented part if you are not using it

Reply:
done